manifests/profile/hathitrust/ingest_jobs.pp

```diff
@@ -0,0 +1,178 @@
+  $joined_dcu_recipients = $dcu_recipients.join(',')
+  $base_env = [
+    "MAILTO=${recipient}",
+    "FEED_HOME=${feed_home}"
+  ]
+
+
+  cron {
+    default:
+      environment => $base_env + [ "HTFEED_CONFIG=${config}" ],
+      user        => 'libadm';
+
+    ## HOURLY  JOBS
+
+    # Pickup rights from CRMS & aleph and populate the rights database
+    'CRMS rights pickup':
+      # TODO: install loadrights script
+      command     => "${feed_perl}/feed.hourly/populate_rights_data.pl --pickup \
+--screen --level INFO 2>&1 >> ${rights_log}",
+      environment => $base_env + [ "HTFEED_CONFIG=${rights_db_config}" ],
+
+      minute      => 25;
+
+    # Queue volumes for conversion on GRIN and check their statuses
+    'GRIN status check':
+      command => "${feed_perl}/feed.hourly/ready_from_grin.pl 2>&1 > /dev/null",
+      minute  => 30;
+
+    ## DAILY JOBS
+
+    # mail previous days' rights load summary
+    'mail rights load summary':
+      command => "/bin/mail -s \"Rights load summary\" ${recipient} < ${rights_log};\
+ mv ${rights_log} ${feed_log}/populate_rights_`date +\"\\%Y\\%m\\%d\"`.log",
+      hour    => 23,
+      minute  => 59;
+
+    # summary of previous day's ingest activity
+    'ingest summary':
+      command => "/bin/bash ${feed_home}/bin/feed.daily/ingest_summary.sh",
+      hour    => 0,
+      minute  => 5;
+
+    # run daily tasks: update grin, get bibrecords, queue new material, deposit rights
+    'daily tasks':
+      command => "for script in ${feed_home}/bin/feed.daily/enabled/*pl; do perl -I${feed_home}/lib \$script; done",
+      hour    => 3,
+      minute  => 0;
+
+    # copy crms renewals from latte-1
+    'crms renewals':
+      command => "/usr/bin/rsync ${crms_renewals_source} ${crms_renewals_dest}",
+      hour    => 4,
+      minute  => 5;
+
+    'copy google rejects list':
+      command     => "${feed_home}/bin/feed.daily/copy_rejects.sh",
+      environment => "MAILTO=${joined_dcu_recipients}",
+      hour        => 3,
+      minute      => 5;
+
+    ## WEEKLY JOBS
+
+    'get brittle books data':
+      command => "${feed_perl}/feed.weekly/get_brittle_books_data.pl 2>&1 > /dev/null",
+      weekday => 1,
+      hour    => 8,
+      minute  => 0;
+
+    'generate ingest logs':
+      command =>  "${feed_perl}/feed.weekly/generate_logs.pl",
+      weekday => 1,
+      hour    => 1,
+      minute  => 0;
+
+    'generate ingest reports':
+      command => "${feed_perl}/feed.weekly/ingest_reporting.pl",
+      weekday => 1,
+      hour    => 2,
+      minute  => 0;
+
+    'repository statistics':
+      command => "/bin/bash ${feed_home}/bin/feed.weekly/repostat.sh",
+      weekday => 1,
+      hour    => 0,
+      minute  => 0;
+
+    'audit statistics':
+      command => "/bin/bash ${feed_home}/bin/feed.weekly/auditstat.sh",
+      weekday => 1,
+      hour    => 0,
+      minute  => 5;
+
+    ## MONTHLY JOBS
+    ########## MONTHLY ############
+
+    # TODO: upload grin_gfv script
```

Comment:
Is this for now or later?

Reply:
Not relevant any more (that was just a wrapper script that set the config for the rights db, which I've inlined)